packages/react-native-bpk-component-calendar/src/NativeCalendar.ios.js

```diff
@@ -59,7 +70,7 @@ class BpkCalendar extends Component<Props, {}> {
       const handle = findNodeHandle(this.calendarRef.current);
       UIManager.dispatchViewManagerCommand(
         handle,
-        UIManager.RCTBPKCalendar.Commands.forceRender,
+        UIManager.getViewManagerConfig('RCTBPKCalendar').Commands.forceRender,
```

Comment:
To remove a deprecation warning. See https://github.com/facebook/react-native/blob/d6236796b285e6ad19c53c5308a0ad9c10792a05/Libraries/ReactNative/UIManagerProperties.js#L15